package.json

```diff
@@ -3,30 +3,35 @@
   "private": true,
   "scripts": {
     "standardjs": "standard",
-    "stylelint": "stylelint **/*.scss",
-    "test": "jest"
+    "stylelint": "stylelint app/assets/stylesheets/**/*.scss",
+    "test": "jest",
+    "build": "esbuild app/javascript/*.* --bundle --sourcemap --outdir=app/assets/builds --public-path=assets --target=es6",
```

Comment:
By default, the "earliest" `esbuild` can transpile to is ES6 JavaScript. Since
IE11 is officially end-of-life and not supported, we took the decision to drop
support for it (i.e ES6 is fine).

If we decide we want to bundle IE11 compatible JS, then we can add an `esbuild`
config file, and use `babel` to transpile compatible code. We might also need to
re-add `regenerator-runtime` for compatible async JS.